engine/swarm/services.md

```diff
@@ -128,15 +135,22 @@ deploys a service to the node. You can apply constraints to the
 service based upon node attributes and metadata or engine metadata. For more
 information on constraints, refer to the `docker service create` [CLI  reference](../reference/commandline/service_create.md).
 
+### Reserving memory or number of CPUs for a service
+
+To reserve a given amount of memory or number of CPUs for a service, use the
+`--reserve-memory` or `--reserve-cpu` flags. If no available nodes can satisfy
+the requirement (for instance, if you request 4 CPUs and no node in the swarm
+has 4 CPUs), the service remains in a pending state until a node is available to
+run its tasks.
```

Comment:
Thanks for adding this. I think someone had pointed this out to me as something that needed more documentation, recently.